multi-int-store/build.gradle

```diff
@@ -1,10 +1,11 @@
 plugins {
 	id "org.springframework.boot" version "2.1.5.RELEASE"
 	id "com.palantir.docker" version "0.22.1"
+	id "io.spring.dependency-management" version "1.0.7.RELEASE"
```

Comment:
This particular plugin has caused a lot of headaches for us in the past (by pulling in non-compatible versions through transitive dependencies). If we think it's fine to bring it back in we should be making this decision for the entire code base instead of individual dependencies.

Reply:
I'll pull it out and see what happens.